ui/src/app/pages/artifactVersion/components/modals/generateClientModal.tsx

```diff
@@ -0,0 +1,293 @@
+                            <FormGroup
+                                label="Client Class Name"
+                                fieldId="form-client-name"
+                            >
+                                <TextInput
+                                    isRequired={true}
+                                    type="text"
+                                    id="form-client-name"
+                                    data-testid="form-client-name"
+                                    name="form-client-name"
+                                    aria-describedby="form-client-name-helper"
+                                    value={this.state.data.clientClassName}
+                                    placeholder="The Class Name to be used"
+                                    onChange={this.onClientNameChange}
+                                />
+                            </FormGroup>
+                        </GridItem>
+
+                        <GridItem span={12}>
+                            <FormGroup
+                                label="Namespace Name"
+                                fieldId="form-namespace"
+                            >
+                                <TextInput
+                                    isRequired={true}
+                                    type="text"
+                                    id="form-namespace"
+                                    data-testid="form-namespace"
+                                    name="form-namespace"
+                                    aria-describedby="form-namespace-helper"
+                                    value={this.state.data.namespaceName}
+                                    placeholder="The Namespace to be used"
+                                    onChange={this.onNamespaceChange}
+                                />
+                            </FormGroup>
+                        </GridItem>
+
+                        <FormGroup
+                            label="Language"
+                            fieldId="form-language"
+                            isRequired={true}
+                        >
+                            <div>
+                                <Dropdown
+                                    toggle={
+                                        <DropdownToggle id="form-type-toggle" data-testid="form-type-toggle" onToggle={this.onLanguageToggle} toggleIndicator={CaretDownIcon}>
+                                            { this.state.data.language ? this.state.data.language : "Java" }
+                                        </DropdownToggle>
+                                    }
+                                    onSelect={this.onLanguageSelect}
+                                    isOpen={this.state.languageIsExpanded}
+                                    dropdownItems={[
+                                        <DropdownItem id="Java" key="Java" data-testid="form-type-auto"><i>Java</i></DropdownItem>,
```

Comment:
may I suggest to sort them alphabetically? Also Kiota has a maturity matrix for languages, you might want to feature that too.

Reply:
done 👍